src/ZombieDriver.php

```diff
@@ -236,7 +236,16 @@ public function setRequestHeader($name, $value)
      */
     public function getResponseHeaders()
     {
-        return (array) $this->server->evalJS('browser.window._response.headers', 'json');
+        $js = <<<JS
+        if (true == zombieVersionCompare('4.0.0', '>=')) {
```

Comment:
could we rely on some feature detection rather than version sniffing ? this would be more consistent with our Zombie 2/3 support